.github/dependabot.yml

```
@@ -61,16 +61,21 @@ updates:
       day: "sunday"
       time: "16:00"
     groups:
-      dependencies:
+      npm-dependencies:
         patterns:
-          - "@dependabot/yarn-lib"
           - "@npmcli/arborist"
-          - "detect-indent"
           - "nock"
           - "npm"
+          - "semver"
+        exclude-patterns:
+          - "detect-indent"
```

```suggestion
          - "detect-indent"  # temp excluded due to https://github.com/dependabot/dependabot-core/pull/5683#issuecomment-1243468605
```

I'd probably prefer to see this one commented...